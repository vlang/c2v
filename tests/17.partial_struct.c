typedef struct TSTexture_t *TSTexture;
