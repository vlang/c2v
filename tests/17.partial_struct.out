[translated]
module main

type TSTexture = voidptr
